modules/es/ut-coverage/es_UT.c

```diff
@@ -3987,6 +4022,13 @@ void TestAPI(void)
     CFE_ES_TaskEntryPoint();
     UtAssert_STUB_COUNT(ES_UT_TaskFunction, 1);
 
+    /* Test deleting a child task when task is not active/valid */
+    ES_ResetUnitTest();
+    ES_UT_SetupSingleAppId(CFE_ES_AppType_EXTERNAL, CFE_ES_AppState_RUNNING, "UT", NULL, &UtTaskRecPtr);
+    TaskId               = CFE_ES_TaskRecordGetID(UtTaskRecPtr);
+    UtTaskRecPtr->TaskId = TaskId + 1; /* UtTaskRecPtr->TaskId shouldn't match the Child Task ID */
```

Comment:
This needs to use the conversion macros - should not directly use IDs as integers (this will fail to compile when strict type checking is enabled, we just keep that off by default for backward compatibility)

Reply:
good catch!  I updated to simply set the TaskId to CFE_ES_TASKID_UNDEFINED. It accomplishes the same goal.